cpu/samd5x/cpu.c

```diff
@@ -49,6 +49,10 @@ static void fdpll0_init(uint32_t f_cpu)
     const uint32_t LDR = ((f_cpu << 5) / 32768);
 
     GCLK->PCHCTRL[OSCCTRL_GCLK_ID_FDPLL0].reg = GCLK_PCHCTRL_GEN(1) | GCLK_PCHCTRL_CHEN;
+    GCLK->PCHCTRL[OSCCTRL_GCLK_ID_FDPLL032K].reg = GCLK_PCHCTRL_GEN(1) | GCLK_PCHCTRL_CHEN;
```

Comment:
From what I understand, this clock is for the lock timer timeout and is optional. Do we really need this ??